primeqa/ir/dense/colbert_top/colbert/modeling/colbert.py

```diff
@@ -28,6 +36,35 @@ def __init__(self, name='bert-base-uncased', colbert_config=None):
         self.query_used = False
         self.doc_used = False
 
+    @classmethod
+    def try_load_torch_extensions(cls, use_gpu):
+        if hasattr(cls, "loaded_extensions") or use_gpu:
+            return
+
+        verbose=os.getenv("COLBERT_LOAD_TORCH_EXTENSION_VERBOSE", "False") == "True"
+
+        print_message(f"Loading segmented_maxsim_cpp extension (set COLBERT_LOAD_TORCH_EXTENSION_VERBOSE=True for more info)...")
+        try:
+            segmented_maxsim_cpp = load(
+                name="segmented_maxsim_cpp",
+                sources=[
+                    os.path.join(
+                        pathlib.Path(__file__).parent.resolve(), "segmented_maxsim.cpp"
+                    ),
+                ],
+                extra_cflags=["-O3"],
+                verbose=verbose,
+            )
+        except (RuntimeError, KeyboardInterrupt) as e:
+            if not verbose:
```

Comment:
Printing stack if_not_ verbose?

Reply:
If we set the verbose flag on the `load` call then I believe it prints the stack regardless of the try/catch

Reply:
OK, makes sense.